examples/doxygen/compressed_block_sparsity_pattern.cc

```diff
@@ -14,7 +14,7 @@
 // ---------------------------------------------------------------------
 
 
-// See documentation of BlockCompressedSparsityPattern for documentation of this example
+// See documentation of BlockDynamicSparsityPattern for documentation of this example
```

Comment:
I think you want to rename this file as well (and find where it is used in the documentation).